lib/nexpose/silo_profile.rb

```diff
@@ -107,7 +107,7 @@ def save(connection)
       begin
         update(connection)
       rescue APIError => error
-        raise error unless (error.message =~ /A silo profile .* does not exist./)
+        raise error unless (error.message =~ /The silo profile .* does not exist/)
```

Comment:
I would agree with @jhart-r7 - might as well just not worry about it again until we have to change it to handle status codes.